frontends/python/prepare_fuzz_imports.py

```
@@ -0,0 +1,193 @@
+                specs = importlib.util.find_spec(_import)
+            except ModuleNotFoundError:
+                continue
+            except ImportError:
+                continue
+            print("No error")
+            if specs is not None:
+                print("Spec:")
+                print(specs)
+                avoid = ['atheris', 'sys', 'os']
+                if _import not in avoid:
+                    if specs.submodule_search_locations:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)